src/app/legacy/components/Footer/index.jsx

```diff
@@ -70,24 +70,48 @@ const StyledAmpCookieSettingsButton = styled(AmpCookieSettingsButton)`
   }
 `;
 
+const showSourcePointMessage = e => {
+  e.preventDefault();
+  if (window.dotcom) {
+    window.dotcom.showSourcePointMessage();
+  }
+};
+
 const SitewideLinks = ({
   links,
   trustProjectLink,
   copyrightText,
   externalLink,
   isAmp,
+  showAdsBasedOnLocation,
   script,
   service,
 }) => {
   const elements = links.map(({ id, text, href, lang }) => {
-    if (isAmp && id === 'COOKIE_SETTINGS') {
-      return (
-        <StyledAmpCookieSettingsButton lang={lang} service={service}>
-          {text}
-        </StyledAmpCookieSettingsButton>
-      );
+    if (id === 'COOKIE_SETTINGS') {
+      if (isAmp) {
+        return (
+          <StyledAmpCookieSettingsButton lang={lang} service={service}>
+            {text}
+          </StyledAmpCookieSettingsButton>
+        );
+      }
+
+      if (showAdsBasedOnLocation) {
+        return (
+          <Link
+            service={service}
+            text={text}
+            href={href}
+            lang={lang}
+            onClick={showSourcePointMessage}
```

Comment:
Looks a good approach overall, the tests need updating here: https://github.com/bbc/simorgh/blob/306d8bab9804138aabe874a45798469a16953a36/src/app/legacy/components/Footer/index.test.jsx#L67